resources/lang/en/allowed_domains.php

```diff
@@ -6,5 +6,5 @@
  */
 
 return [
-    'meltwater.org'
+    'gmail.com'
```

Comment:
Please fix this

Reply:
Are you asking me to add "meltwater.org" to the allowed domains?